src/Core/Curve25519/Fe.php

```diff
@@ -104,7 +104,7 @@ public function offsetUnset($offset): void
      * @psalm-suppress ImplementedReturnTypeMismatch
      */
     #[ReturnTypeWillChange]
```

Comment:
What with the PHP minimum having been raised to PHP 8.1+, are the `#[ReturnTypeWillChange]` attributes still needed ? I'd think those can be removed now (and the return type added).

Reply:
Good question. We'll look at this in the future.